playwright/tests/treeTestResultsTests.spec.js

```diff
@@ -61,4 +61,231 @@ test('Unique link results page test', async ({ studiesPage, resultsPage, editTre
     //Delete Studuies
     await studiesPage.goto();
     await studiesPage.deleteStudy(testTitle);
+});
+
+test('Unique link give up results page test', async ({ studiesPage, resultsPage, editTreeTestPage, treeTestStudy}) => {
```

Comment:
Would these not be best in the uniqueLink.spec file? or if these are the big e2e scenarios have a e2e.spec file of their own?

Reply:
I think they should remain in the results page, as it is ultimately the final check that this has worked.